tools/swift-ide-test/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ if(LibXml2_FOUND)
   include_directories(SYSTEM ${LIBXML2_INCLUDE_DIR})
   target_link_libraries(swift-ide-test PRIVATE ${LIBXML2_LIBRARIES})
   target_compile_definitions(swift-ide-test PRIVATE SWIFT_HAVE_LIBXML=1)
+  if("${CMAKE_SYSTEM_NAME}" STREQUAL "OpenBSD")
+    include_directories(SYSTEM "/usr/local/include")
+  endif()
```

Comment:
Is `/usr/local/include` not in the default include path on OpenBSD toolchains?  This should at least be `CMAKE_HOST_SYSTEM_NAME` and `NOT CROSS_COMPILING`.

Reply:
Indeed, it is not.

Amended to use `CMAKE_HOST_SYSTEM_NAME` and `NOT CMAKE_CROSSCOMPILING`.